tests/unit/properties/non_scalar/test_symmetry.py

```diff
@@ -21,5 +25,6 @@ def tearDown(self):
 
     def test_symmetry(self):
         parser = self.get_mocked_parser("space_group_symbol", SPACE_GROUP_SYMBOL)
+        parser.attach_mock(MagicMock(return_value=POINT_GROUP_SYMBOL), "point_group_symbol")
```

Comment:
The unit test was failing because no value for `point_group_symbol` was being created within the magicMock object, and because the values for point group and space group are created separately there was not a good way to add both values to a magicMock object at the same time using our `get_mocked_parser` function.
The original error I was getting is below.

```jsonschema.exceptions.ValidationError: <MagicMock name='mock.space_group_symbol().__getitem__()' id='4744938016'> is not of type 'string'

Failed validating 'type' in schema['properties']['spaceGroupSymbol']:
    {'description': 'space group symbol in Hermann–Mauguin notation',
     'type': 'string'}

On instance['spaceGroupSymbol']:
    <MagicMock name='mock.space_group_symbol().__getitem__()' id='4744938016'>```

So after chatting with @tjduigna adding in a line to attach the point group symbol value to the magicMock object was the easiest solution to fix the failing test.